index.html

```diff
@@ -1,11 +1,757 @@
+            large-scale structures of the universe.
+          </div>
+          <div class="theory-down t3">
+            <span>FLUCTUATION</span> Astronomers have now detected evidence of
+            these ancient fluctuations in swirls of polarized light in the
+            cosmic background radiation, which is energy left over from the
+            early universe. These are gravitational waves predicted by Einstein.
+          </div>
+          <div class="theory-down t4">
+            <span>EXPANSION</span> Returning to our coffee, imagine a single,
+            central pot expanding faster than light and cooling to an even
+            temperature as it expands. That is something like inflation. And the
+            structure of the universe mirrors the froth and foam of the original
+            pot.
+          </div>
+        </div>
+
+        <div class="credits">
+          <hr>
+          <p>By LARRY BUCHANAN and JONATHAN CORUM</p>
+        </div>
+
+        <div class="author content font">
+          <p>
+            Dr. Kovac said the chance that the results were a fluke was only one
+            in 10 million.
+          </p>
+          <p>
+            Dr. Guth, now 67, pronounced himself “bowled over,” saying he had
+            not expected such a definite confirmation in his lifetime.
+          </p>
+          <p>
+            “With nature, you have to be lucky,” he said. “Apparently we have
+            been lucky.”
+          </p>
+          <p>
+            The results are the closely guarded distillation of three years’
+            worth of observations and analysis. Eschewing email for fear of a
+            leak, Dr. Kovac personally delivered drafts of his work to a select
+            few, meeting with Dr. Guth, who is now a professor at Massachusetts
+            Institute of Technology (as is his son, Larry, who was sleeping that
+            night in 1979), in his office last week.
+          </p>
+          <p>
+            “It was a very special moment, and one we took very seriously as
+            scientists,” said Dr. Kovac, who chose his words as carefully as he
+            tended his radio telescopes.
+          </p>
+          <p>
+            Andrei Linde of Stanford, a prolific theorist who first described
+            the most popular variant of inflation, known as chaotic inflation,
+            in 1983, was about to go on vacation in the Caribbean last week when
+            Chao-Lin Kuo, a Stanford colleague and a member of Dr. Kovac’s team,
+            knocked on his door with a bottle of Champagne to tell him the news.
+          </p>
+          <iframe
+            width="560"
+            height="315"
+            src="https://www.youtube.com/embed/ZlfIVEy_YOA"
+            frameborder="0"
+            allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
+            allowfullscreen>
+          </iframe>
```

Comment:
`REQUIRED`

Images and videos should be enclosed by a `<figure>` tag with `<figcaption>` that gives a description of the embedded image or video